test/list_test.go

```diff
@@ -0,0 +1,75 @@
+package test
+
+import (
+	"context"
+	"fmt"
+	"testing"
+
+	. "github.com/onsi/gomega"
+	clientv3 "go.etcd.io/etcd/client/v3"
+)
+
+// TestList is the unit test for List operation.
+func TestList(t *testing.T) {
+	ctx := context.Background()
+	client := newKine(t)
+
+	t.Run("ListSuccess", func(t *testing.T) {
```

Comment:
We should have tests that return data. We should also make sure we go through the execution paths of Get, Range all the way to List and ListCurrent.

Reply:
Done!